src/main/resources/application.properties

```diff
@@ -3,6 +3,52 @@ DOMAIN=localdomain
 OAUTH_CLIENT_ID=null
 OAUTH_SECRET=null
 
+SKIP_SSL_VALIDATION="true"
+SPRING_PROFILES_ACTIVE=cloud,pki-geoserver-auth
+
+# The location of the deployed resource server sample application
+# RESOURCE_URL=https://russell-sample.dev.dev.east.paas.geointservices.io
+
+# Grant type to be set for the application's UAA client configurations. Only one grant type per application is supported by the SSO service.
+GRANT_TYPE=authorization_code
+
+# Identity provider(s) to be set for the application's client configurations
+SSO_IDENTITY_PROVIDERS=uaa
+
+# The following are bootstrap configurations you may use to automatically create client configurations in the SSO service for your application if the configurations do not exist. These configurations take effect when binding or rebinding to the SSO, and will overwrite existing client configurations if any. The values provided below are examples.
+
+# Whitelist of redirect URI(s) allowed for the application. This value must start with http:// or https://
+# SSO_REDIRECT_URIS=https://my-domain-here.domain.org
```

Comment:
For this, and all other parameters that are commented out in this file, let's remove them. Let's keep only what we need and not clutter things up.

Reply:
Done.

Reply:
We can agree to disagree. But if we want to keep them, can we at least move them to a different file?  Or do they have to be within application.properties?